### ISSUE TITLE
IOException on start-up

### ISSUE THREAD

**Issue by SlimeDog** [PyroTempus/My-Plugins#40]
Spigot 1.13.2 final and 1.14.3 latest
PWP 3.2.0

On server start-up, the console/log displays
```
[22:15:01] [Server thread/INFO]: [PyroWeatherPro] Enabling PyroWeatherPro v3.2.0
[22:15:01] [Server thread/INFO]: [PyroWeatherPro] Successfully hooked into WorldGuard.
[22:15:01] [Server thread/INFO]: [PyroWeatherPro] GriefPrevention not found, skipping.
[22:15:01] [Server thread/WARN]: java.io.IOException: Connect failed
[22:15:01] [Server thread/WARN]:        at me.lucko.networkinterceptor.interceptors.ProxySelectorInterceptor$LoggingSelector.select(ProxySelectorInterceptor.java:59)
[22:15:01] [Server thread/WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1175)
[22:15:01] [Server thread/WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1065)
[22:15:01] [Server thread/WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1063)
[22:15:01] [Server thread/WARN]:        at java.base/java.security.AccessController.doPrivileged(Native Method)
[22:15:01] [Server thread/WARN]:        at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:793)
[22:15:01] [Server thread/WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1062)
[22:15:01] [Server thread/WARN]:        at java.base/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
[22:15:01] [Server thread/WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1581)
[22:15:01] [Server thread/WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1501)
[22:15:01] [Server thread/WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1499)
[22:15:01] [Server thread/WARN]:        at java.base/java.security.AccessController.doPrivileged(Native Method)
[22:15:01] [Server thread/WARN]:        at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:793)
[22:15:01] [Server thread/WARN]:        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1498)
[22:15:01] [Server thread/WARN]:        at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:245)
[22:15:01] [Server thread/WARN]:        at me.arsmagica.Listeners.DisableWeather.a(DisableWeather.java:63)
[22:15:01] [Server thread/WARN]:        at me.arsmagica.PyroWeather.onEnable(PyroWeather.java:121)
[22:15:01] [Server thread/WARN]:        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:265)
[22:15:01] [Server thread/WARN]:        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:347)
[22:15:01] [Server thread/WARN]:        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:410)
[22:15:01] [Server thread/WARN]:        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugin(CraftServer.java:436)
[22:15:01] [Server thread/WARN]:        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugins(CraftServer.java:350)
[22:15:01] [Server thread/WARN]:        at net.minecraft.server.v1_13_R2.MinecraftServer.l(MinecraftServer.java:580)
[22:15:01] [Server thread/WARN]:        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:542)
[22:15:01] [Server thread/WARN]:        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:420)
[22:15:01] [Server thread/WARN]:        at net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:294)
[22:15:01] [Server thread/WARN]:        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:698)
[22:15:01] [Server thread/WARN]:        at java.base/java.lang.Thread.run(Thread.java:834)
```
I imagine that PWP is trying to connect to check for updates. I do not want it to do that (I block outgoing connections, and have my own automagical update checker). But I do not find a configuration option to disable it. Please do the needful.

**Comment by SlimeDog** [comment]
This issue still exists in PWP 3.3.0.

**Comment by PyroTempus** [comment]
Fixed for v3.3.1
